config/application.rb

```diff
@@ -23,5 +26,9 @@ class Application < Rails::Application
     config.autoload_paths += %W(#{config.root}/lib)
 
     config.assets.precompile += %w(comfortable_mexican_sofa/admin/basic.css rsvp/rsvp.js)
+
+    if Feature.active?(:redirects)
+      config.middleware.use Rack::RedirectMiddleware::Responder
+    end
```

Comment:
the features toggles are served in `config/features/ENV.yml`, so from the checked in code base and not scripts